HW06.Rmd

```diff
@@ -14,21 +16,44 @@ Steps:
   3. For 27 genes of interest AND your favorite gene, perform a t-test to see if it is differentially expressed between the WT vs H3K27M samples
   4. Create a graphing function and then create a boxplot that graphs expression between the two groups
   
-  
-#### Code
+```{r setup, echo = FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(knitr)
+library(dplyr)
+### Preparing the packages I'll need
+```
 
 From the RNA-Seq files, you only need the "Name" from one file and the "TPM" column from all the files. TPM stands for "transcripts per million" and is a common unit for normalized expression data. 
-```{r load in data}
-#hint, using apply (specifically sapply) you can read in the data into a list and then bind the columns together. Or you can use a for loop too. 
-
-#you only need the 
 
+```{r load in data}
+WT_names <- list.files("./RNA_Seq_processed/WT", full.names=TRUE)
+H3K27M_names <- list.files("./RNA_Seq_processed/H3K27M", full.names=TRUE)
+
+WT_files_all <- lapply(WT_names, read_tsv)
+WT_files <- lapply(WT_files_all, select, contains(c("Name", "TPM")))
+for (i in 1: length(WT_files))
+  {colnames(WT_files[[i]]) <- c("gene_id", paste0("TPM_", i, "_WT"))}

```

Comment:
AH! I think this is what I was missing in my code, a way to identify the two separate groups down the line.  I like how you added this to their column name.  I was just combining the two groups and forgot to add in a way to know where they came from so I think this is where my issues were coming from.